index.html

```diff
@@ -6558,6 +6558,16 @@ <h3 id="Convolution-channel-configurations">
               thrown</span>. The default value is 2048. Note that large FFT
               sizes can be costly to compute.
             </p>
+            <p>
+              If the <code>fftSize</code> is changed to a different value, then
+              all state associated with smoothing of the frequency data (for
```

Comment:
Can we spell out the state that is reset here? We can link to https://webaudio.github.io/web-audio-api/#dfn-previous-block I think.